### ISSUE TITLE
Query Related to Number of Epoch and Loss

### ISSUE THREAD

**Issue by Zain-Gill123** [seominseok0429/Real-world-Anomaly-Detection-in-Surveillance-Videos-pytorch#7]
Hello! I am new to python and NN. I wanted to know for how many **EPOCH** should I train this model? Also, is there any limit to how low the **LOSS** can be? And should one stop training if the **LOSS** is not reducing? Any other information a newbie should have regarding the **LOSS VALUE** and the **EPOCH**?

Any type of comments would be highly appreciated! Cheers!

-Zain

**Comment by seominseok0429** [comment]
You only need to run 75 epochs. This code stores the highest AUC for 75 epochs.
